corehq/apps/userreports/management/commands/async_rebuild_table.py

```diff
@@ -93,16 +98,25 @@ def _get_ids_to_process(self):
 
     def _get_ids(self, db):
         if self.referenced_type == CASE_DOC_TYPE:
-            return (
+            date_column = 'modified_on'
+            query = (
                 CommCareCaseSQL.objects
                 .using(db)
                 .filter(domain=self.domain, type=self.case_type_or_xmlns)
                 .values_list('case_id', flat=True)
             )
         elif self.referenced_type == XFORM_DOC_TYPE:
-            return(
+            date_column = 'received_on'
+            query = (
                 XFormInstanceSQL.objects
                 .using(db)
                 .filter(domain=self.domain, xmlns=self.case_type_or_xmlns)
                 .values_list('form_id', flat=True)
             )
+        if self.start:
+            kw = '{}__gte'.format(date_column)
+            query = query.filter(kw=self.start)
```

Comment:
I think to do this you need to do `**{kw: self.start}`

Reply:
you could also do this with `Q` objects